build.sbt

```diff
@@ -546,7 +546,33 @@ lazy val scioTensorFlow: Project = Project(
     "io.circe" %% "circe-core",
     "io.circe" %% "circe-generic",
     "io.circe" %% "circe-parser"
-  ).map(_ % circeVersion)
+  ).map(_ % circeVersion),
+  (testLoader in Test) := {
```

Comment:
@jto awesome! Is it possible to put this in a separate file say in project dir and import - for style sake, this file is already uber complex, and this just adds fuel to the fire. What do you think?

Reply:
👍 on moving to separate file if possible 😄

Reply:
@ravwojdyla @regadas done :)